src/components/Capture/Face.js

```diff
@@ -79,7 +86,7 @@ class Face extends Component {
           onCapture={ this.handleCapture }
         />
       :
-      this.props.uploadFallback ?
+      props.uploadFallback ?
```

Comment:
Not strictly related to this PR - but this is a bit of a weird pattern, here. This `props` was created just to introduce a default prop of `onError: this.handleError`

Default props have a clear pattern already in React - doing this weird `props` thing instead is just confusing

Reply:
You are right, it's ugly. I will change that as part of another ticket and make sure the pattern is not used elsewhere